android/app/src/main/AndroidManifest.xml

```diff
@@ -30,6 +30,10 @@
         <meta-data
             android:name="flutterEmbedding"
             android:value="2" />
+        <meta-data
+            android:name="com.google.android.geo.API_KEY"
+            android:value="AIzaSyBp0pAND5nl9u0YanI9W11MXYTVLs5FoX0" />
```

Comment:
is this supposed to be public ? where did you find it ? is it tied to your personal google account ? this should be documented.